acceptance/bundle/python/resource-ordering/databricks.yml

```diff
@@ -0,0 +1,15 @@
+bundle:
+  name: my_project
+
+sync: {} # dont need to copy files
```

Comment:
This doesn't have the expected effect. You have to configure the `paths` field to an empty list.